docs/fastdds/xml_configuration/common.rst

```diff
@@ -0,0 +1,416 @@
+| ``<propagate>`` | Indicates if it is going to be serialized along with the |br|       | ``Boolean`` | ``false``      |
+|                 | object it belongs to.                                               |             |                |
++-----------------+---------------------------------------------------------------------+-------------+----------------+
+
+**Example**
+
+.. literalinclude:: /../code/XMLTester.xml
+    :language: xml
+    :start-after: <!-->XML-PROPERTIES-POLICY<-->
+    :end-before: <!--><-->
+
+.. _DurationType:
+
+DurationType
+^^^^^^^^^^^^
+
+DurationType expresses a period of time and it is commonly used inside other XML elements, such as in
+``<leaseAnnouncement>`` or ``<leaseDuration>``.
+A DurationType is defined by two mandatory elements ``<sec>`` plus ``<nanosec>``.
+An infinite value can be specified by using the values :cpp:concept:`DURATION_INFINITY`,
+:cpp:concept:`DURATION_INFINITE_SEC` and :cpp:concept:`DURATION_INFINITE_NSEC`.
+
++-----------------------+---------------------------------------------------------+-------------------+----------------+
+| Name                  | Description                                             | Values            | Default        |
++=======================+=========================================================+===================+================+
+| ``<sec>``             | Number of seconds.                                      | ``Int32``         | 0              |
++-----------------------+---------------------------------------------------------+-------------------+----------------+
+| ``<nanosec>``         | Number of nanoseconds.                                  | ``UInt32``        | 0              |
++-----------------------+---------------------------------------------------------+-------------------+----------------+
+
+**Example**
+
+.. literalinclude:: /../code/XMLTester.xml
+    :language: xml
+    :start-after: <!-->XML-DURATION<-->
+    :end-before: <!--><-->
+
+.. _TopicType:
+
+TopicType
+^^^^^^^^^
+
+The |Topic| name and data type are used to determine whether Datawriters and DataReader can exchange messages.
+Please refer to :ref:`dds_layer_topic` section for a a deeper explanation on the |Topic| class.
+
++-------------------------+-----------------------------------------------+--------------------------+-----------------+
+| Name                    | Description                                   | Values                   | Default         |
++=========================+===============================================+==========================+=================+
+| ``<kind>``              | It defines the Topic's key kind. |br|         | :class:`NO_KEY`          | :class:`NO_KEY` |
+|                         | See :ref:`dds_layer_definition_data_types`.   | :class:`WITH_KEY`        |                 |
++-------------------------+-----------------------------------------------+--------------------------+-----------------+
```

Comment:
The `<kind>` is not used in DDS, it is deduced automatically from the dataType (this field would be ignored).
But it is still used if the RTPS-API is used. So I don't know if we should leave it o not.